### PR TITLE
Mex

### PR DESCRIPTION
**Pull request by matittner**
I've noticed a huge area without airports in northern Mexico and found out all these cities are missing.
It also adds the missing airport requested on the forum and a nearby 100k city.

### CODE REVIEW

**Comment by patsonluk** [comment]
will probably merge this in tomorrow thanks!

**Comment by patsonluk** [comment]
@matittner are u going to add more or should i merge these and u open more PRs later?

**Comment by matittner** [comment]
I'm a little busy until xmas, so I don't think I'll add more stuff for about another 1-2w, but after that I plan on adding more missing stuff as discussed in the forum.
So, it is your call, smaller incremental changes or a big one later on, either are fine by me.

File: airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala [hunk 1]
```diff
@@ -73,6 +73,7 @@ object AirportFeaturePatcher {
         "CUN" -> 32, //Cancun
         "ORY" -> 32, //Paris Orly
         "GUM" -> 31, //Guam
+        "BDA" -> 35, //Bermuda
```
Comment (patsonluk) [inline airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala]: 35 seems way too strong, i dun think it should be more than Cancun and Guam...

Not really sure how big of a vacation spot is Bermuda, I searched Hotels.com and Bermuda gives like 35 result, while other countries in Carribean, for example Aruba - gives 288 , while Aruba i only gave it a 11 charm and Barbados - 382 hotels, which is only with 11 charm as well :)

Comment (matittner) [reply airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala]: I didn't think too much about it, I saw Tenerife, the Canary islands, Pape'ete (no idea where's that), Punta Cana (no idea either) and figured Bermuda would be around the same ballpark, I didn't know you had a metric like that. 
So, ~8?

File: airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala [hunk 2]
```diff
@@ -146,7 +146,8 @@ object AirportFeaturePatcher {
         "TAO" -> 10, //Qingdao China
         "GEA" -> 10,
         "YZF" -> 10, //Yellowknife
-        "CUR" -> 10 //Curacao
+        "CUR" -> 10 //Curacao      
+        "BDA" -> 10, //Bermuda
```
[the remark this answers is not in the document]
Comment (matittner) [reply airline-data/src/main/scala/com/patson/init/AirportFeaturePatcher.scala]: Then we shouldn't give it anything, I thought it was more popular. 
I forgot to add the comma when sorting, sorry.

**Comment by patsonluk** [comment]
Thank you so much @matittner  

Merging!

**Pull request merged**